examples/models/toolbars2.py

```diff
@@ -1,3 +1,11 @@
+''' This example shows multiple ways to place the toolbar with respect to a grid of up to four plots.
```

Comment:
This example happens to have four plots, but there is nothing special about that number, these locations will work with any grid size.
```suggestion
''' This example shows multiple ways to place the toolbar with respect to a grid of plots.
```